docs/sources/writing-guide/front-matter/index.md

```diff
@@ -44,14 +44,12 @@ The following headings describe what each element does and provides guidelines f
 
 ### aliases
 
-Provides an HTML redirect from the pages in the list to the current page.
-For more information, refer to [Hugo aliases](#hugo-aliases).
+Used to create redirects from the previous URL to the new URL when a page changes or moves. As a best practice, when you rename or move files, you should create an alias with the previous file path and name to create an HTML redirect from the old URL to the new URL. For more information, refer to [Hugo aliases](#hugo-aliases).
 
-### description
+### (required) description
```

Comment:
Not sure if the "required" should come after the item rather than before. I was gonna see what it looked like in preview, but you need to merge your branch with main to get the updates that allow the make docs command to work.

Reply:
My aim was to have "required" visible in the right TOC.  I don't really care if it's before or after the heading, but I've usually seen the "required" or "optional" at the beginning so that people who scan don't miss it, and if things wrap it doesn't get lost.

I couldn't get `make-docs` to run last week, so I haven't seen how this looks in a preview yet.  I should check and see if Jack fixed the script.

Reply:
He has! I think you might have to merge your branch with main to get the fix.

Reply:
Yeah looking at this in a local build, I find the list less scannable seeing the word in parentheses first and then the actual thing element i want information about second. recommend moving (required) to after the front matter element.
```suggestion
### description (required) 
```